arangod/Agency/FailedLeader.cpp

```diff
@@ -302,6 +301,13 @@ bool FailedLeader::start(bool& aborts) {
         addPreconditionServerHealth(pending, _to, "GOOD");
         // Server list in plan still as before
         addPreconditionUnchanged(pending, planPath, planned);
+        // Server list in Current still as known
+        addPreconditionUnchanged(pending, curPath + "/servers", current);
+        // Failover candidates in Current still as known
+        auto const& failoverCandidates = _snapshot.hasAsSlice(curPath + "/failoverCandidates");
+        if (failoverCandidates.second) {
```

Comment:
This is good, if it is not present we do not check for it. This might trigger during upgrades.